requirements.txt

```diff
@@ -7,7 +7,7 @@
 # lines are added as extras under a group called "<tag>".
 
 numpy >= 1.19
-aiohttp >= 3.7.4.post0
+aiohttp >= 3.9.0b0
```

Comment:
we should check against the rc0 they have

Reply:
Sure they have that too.